### DISCUSSION TITLE
国内实在没法用，请问有其他版本吗？

### DISCUSSION THREAD

**Discussion by abc62201** [spencerwooo/onedrive-vercel-index#822] [General]
我有一个虚拟空间，就是人家服务器上的一个文件夹（便宜）
这种情况如何使用这个软件？

**Comment by wdw1007** [comment]
因为 vercel.app 被墙所以连接不了
可以考虑购买一个域名或者申请一个免费域名

// 亦或者使用 netlify 搭建, 目前他们提供的域名没有受到阻断
// 废了 现在提示 "Task timed out after 10.02 seconds"

**Reply by steady298** [reply]
用自己域名可以进网站 但是还是无法下载

**Reply by wdw1007** [reply]
我通过 cf cname 到 cname.vercel-dns.com 的站点没有问题欸
ovi.wdw1007.eu.org
![图片](https://github.com/spencerwooo/onedrive-vercel-index/assets/87262434/e780c610-9ec5-47b5-ba00-7b464f7149df)

**Reply by BrandonStudio** [reply]
@steady298 你用的是OneDrive个人版吗？那样的话被屏蔽的是OneDrive本身

**Reply by steady298** [reply]
那什么版本不会被屏蔽呢> @steady298 你用的是OneDrive个人版吗？那样的话被屏蔽的是OneDrive本身

**Reply by BrandonStudio** [reply]
@steady298 企业版不会被屏蔽。企业版使用 *-my.sharepoint.com域名，而不是onedrive.com或onedrive.live.com

**Reply by steady298** [reply]
好的 感谢回复> @steady298 企业版不会被屏蔽。企业版使用 *-my.sharepoint.com域名，而不是onedrive.com或onedrive.live.com
